src/main/scala/tile/FPU.scala

```diff
@@ -727,7 +727,9 @@ class FPU(cfg: FPUParams)(implicit p: Parameters) extends FPUModule()(p) {
   val io = IO(new FPUIO)
 
   val (useClockGating, useDebugROB) = coreParams match {
-    case r: RocketCoreParams => (r.clockGate, r.debugROB)
+    case r: RocketCoreParams =>
+      val sz = if (r.debugROB.isDefined) r.debugROB.get.size else 1
+      (r.clockGate, sz < 1)
```

Comment:
Hmmm, this means that there is no write tracking of writes into FPRs right?

Reply:
Yep. Just integer registers for now.

Reply:
Ok then